benchmarks/tasks.py

```diff
@@ -23,7 +23,13 @@
 
 
 @celery_app.task(bind=True)
-def set_testjob_results(self, testjob):
+def set_testjob_results(self, testjob_id):
+    if type(testjob_id) in (str, unicode):
+        testjob = models.TestJob.objects.get(pk=testjob_id)
+    else:
+        # TODO backwards compatibility for jobs already queued before upgrade.
```

Comment:
I'd say let's remove it already now. No need to preserve backward compatibility in this case as this is internal API and the jobs are scheduled periodically. During the upgrade (this time) we can stop the queue and remove all jobs manually.